sources/packages/web/src/services/http/ApplicationApi.ts

```diff
@@ -26,4 +26,17 @@ export class ApplicationApi extends HttpBaseClient {
       throw error;
     }
   }
+
+  public async getNOA(applicationId: any): Promise<any> {
```

Comment:
When using an acronym please consider adding some comments that just describe of the acronym means.
For instance:
```
/**
   * Retrieve the Notice of Assessment (NOA) for a particular application.
   */
```